Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.2-fpm
+FROM php:7.3-fpm
```

Comment:
@nicolas-bastien actually if i php --version on the latest pull of smartbooster/php-fpm:builder on smartpilot i still get this

> PHP 7.2.10 (cli) (built: Sep 15 2018 02:33:49) ( NTS )
> Copyright (c) 1997-2018 The PHP Group
> Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
>     with Zend OPcache v7.2.10, Copyright (c) 1999-2018, by Zend Technologies

Reply:
@mathieu-ducrot I added a ssh command to validate and it's ok for me

![Capture du 2019-12-29 08-22-51](https://user-images.githubusercontent.com/1446466/71553736-da653600-2a14-11ea-9c19-807340ee1cfc.png)

I checked if I miss to push but it was ok, can you check ?

Reply:
@nicolas-bastien it's ok for the latest version but check it on the builder tag version : 

`docker run --rm -it smartbooster/php-fpm:builder bash` it prompt me a PHP 7.2.10

Reply:
@mathieu-ducrot ok I've fixed it in next PR !